lib/rails/generators/neo4j/install/templates/neo4j_rails5.rb

```diff
@@ -0,0 +1,9 @@
+# Configure NEO4J as the ORM
+Rails.application.config.generators do |g|
+  g.orm             :neo4j
+end
+
+# Configure where the neo4j database should exist
+Neo4j::Config.use do |config|
+  config[:storage_path] = "#{Rails.application.config.root}/db/neo4j-#{Rails.env}"
```

Comment:
Yeah, agreed about convention.  `http://localhost:7474` is actually the default in further down in `neo4j-core`, but I'm not sure what it should be for embedded.  I actually just realized I was going to mention before that we already have a convention for how the rake tasks create databases (for server mode, though maybe it could be shared behavior) in:

    db/neo4j/environment